proposals/20210525-feast_example_gen.md

```diff
@@ -0,0 +1,75 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+**Your name:** Badrul Chowdhury
+
+**Your email:** badrulchowdhury17@gmail.com
+
+**Your company/organization:** Apple
+
+**Project name:** ExampleGen for Feast Feature Store
+
+## Project Description
+
+This proposal is for a new component to enable connecting to and reading from feature stores via a simple and unified API. The first iteration will only support Feast [1], but we will add support for other popular feature stores over time. Similarly, we will restrict v1 of our library to only read from feature stores- we will evaluate supporting writes later.
```

Comment:
Probably it's out of scope for now, but it will be great to have a similar integration to read from managed feature store on GCP (Vertex AI Feature store) @rcrowe-google

Reply:
is Vertex AI Feature store based on FEAST?

Reply:
> is Vertex AI Feature Store based on FEAST?

It does not seem to be.

Reply:
> Probably it's out of scope for now, but it will be great to have a similar integration to read from managed feature store on GCP (Vertex AI Feature store) @rcrowe-google

We're working on making it really easy to add new backends to Feast as `providers`. Adding support for the GCP feature store as an underlying implementation is also possible at the Feast level.